hedera-node/hedera-network-admin-service-impl/src/main/java/com/hedera/node/app/service/networkadmin/impl/handlers/FreezeHandler.java

```diff
@@ -86,7 +86,8 @@ public void preHandle(@NonNull final PreHandleContext context) throws PreCheckEx
         }
         if (Arrays.asList(FREEZE_UPGRADE, TELEMETRY_UPGRADE, PREPARE_UPGRADE).contains(freezeType)) {
             // from proto specs, it looks like updateFileId not required for FREEZE_UPGRADE and TELEMETRY_UPGRADE
-            // but specs aren't very clear previous code in FreezeTransitionLogic checks for it in all 3 cases,
+            // but specs aren't very clear
+            // previous code in FreezeTransitionLogic checks for it in all 3 cases
```

Comment:
I guess this can get streamlined

Reply:
Not sure where this came in, it can be undone if desired, however.

Reply:
I reverted it